src/main/java/com/cairn/waypoint/dashboard/endpoints/HomeworkQuestionLinkedProtocolTemplate/HomeworkQuestionLinkedProtocolTemplateDto.java

```diff
@@ -0,0 +1,12 @@
+package com.cairn.waypoint.dashboard.endpoints.HomeworkQuestionLinkedProtocolTemplate;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
```

Comment:
Kind of like you've done here with @Data. This will generate your getters/setters/etc